src/main/scala/org/hammerlab/guacamole/DistributedUtil.scala

```diff
@@ -336,8 +350,11 @@ object DistributedUtil extends Logging {
     readsRDDs: PerSample[RDD[MappedRead]],
     lociPartitions: LociMap[Long],
     skipEmpty: Boolean,
-    function: PerSample[Pileup] => Iterator[T],
+    function: (PerSample[Pileup], ContigSequence) => Iterator[T],
```

Comment:
Since each `Pileup` has a `ContigSequence` attribute, the user defined function already has access to the contig sequence, so I'm not sure it makes sense to add this argument